t-metric.js

```diff
@@ -0,0 +1,146 @@
+"use strict"
+const tmetricToken = process.env.HUBOT_TMETRIC_TOKEN
+const tmetricAccountId = process.env.HUBOT_TMETRIC_ACCOUNT_ID
+const tmetricClientList = process.env.HUBOT_TMETRIC_CLIENT_LIST
+const room = 'developers'
+var cron = require('node-cron');
+
+
+module.exports = (robot) => {
+  robot.error(function (err, res) {
+    robot.logger.error(err)
+    robot.send({room: room}, `there is an issue with the t-metric bot '${err}'`)
+  })
+  robot.hear(/tmetric\smonthly/gi, (res) => {
+    tmetricMonthlyReport(
+      robot,
+      {
+        res: res
+      }
+    )
+  })
+
+  robot.hear(/tmetric\sweekly/gi, (res) => {
+    tmetricWeeklyReport(
+      robot,
+      {
+        res: res
+      }
+    )
+  })
+
+  cron.schedule('30 16 * * 5', () => {
+    tmetricWeeklyReport(
+      robot,
+      {
+        outputPrefix: '@all ',
+        outputPostfix: 'Aaphno :tea:-metric :thumbsup:-sanga :straight_ruler:-bhayo?'
+      },
+    )
+  })
+
+  cron.schedule('00 16 31 * *', () => {
```

Comment:
what about the 30th of April...?

I don't know if something like this can be done: https://www.nexcess.net/blog/cron-job-for-the-last-day-of-the-month/

Or maybe:
```suggestion
  cron.schedule('00 16 28-31 * *', () => {
```

And in `tmetricMonthlyReport` put some code that gets the date of tomorrow="today+1". And tomorrow's date is the first of the month, then run the bot job, otherwise do nothing.

Reply:
good idea, implemented in afd94ecce8489c50adda974dde7fd734a3a06007